support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx

```diff
@@ -228,22 +234,20 @@ export function SupporterPlusThankYou({
 	);
 
 	const maybeThankYouModule = (
-		condtion: boolean,
+		condition: boolean,
 		moduleType: ThankYouModuleType,
-	): ThankYouModuleType[] => (condtion ? [moduleType] : []);
-
-	const showFeast =
-		!!abParticipations.feast &&
-		useContributionsSelector(isSupporterPlusFromState);
+	): ThankYouModuleType[] => (condition ? [moduleType] : []);
 
+	// abParticipation, upon refresh, defaults to active abtTests only
```

Comment:
I don't really understand this comment, could you explain a little more?

Reply:
If you reload the thankyou page, only ab-tests that are marked as `isActive=true` run (including `feast` abtest, which means the Feast card will hide, when the ab-test goes live (ie `isActive=true`) the Feast card will remain visible)